src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/AppointmentIntegrationTest.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.whereabouts.integration
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDate
+
+@Transactional
+class AppointmentIntegrationTest : IntegrationTest() {
+
+  @Autowired
+  lateinit var jdbcTemplate: JdbcTemplate
+
+  @BeforeEach
+  fun prepare() {
```

Comment:
Copy paste?

Reply:
Yes, quite a few of these in this class. I'll revisit